dbfit-java/core/src/main/java/dbfit/api/vendor/DbStoredProcedureCall.java

```diff
@@ -0,0 +1,16 @@
+package dbfit.api.vendor;
+
+import java.sql.SQLException;
+
+import dbfit.fixture.StatementExecution;
+
+public interface DbStoredProcedureCall {
+
+    String getName();
+
+    boolean isFunction();
```

Comment:
What we decide here? Maybe we can drop the redundant functions from the interface and see if we'd like to drop the whole interface or to keep it as subsequent step?